cmd/nvkind/cluster_create.go

```diff
@@ -95,6 +106,34 @@ func BuildClusterCreateCommand() *cli.Command {
 	return &cmd
 }
 
+func runChildProcess(args []string, input string) error {
+	log.Printf("nvkind: run: %v\n", args)
+	cmd := exec.Command(args[0], args[1:]...)
+
+	// Let child's stdout/err flow directly to current process's standard streams.
```

Comment:
I typed "process's" intuitively. Now it looked funky. Relief: https://forum.wordreference.com/threads/process-or-processs.1704502/